docs/Unused-Private-Method.md

```diff
@@ -0,0 +1,46 @@
+## Introduction
+
+Classes should use their private methods. Otherwise this is dead
+code which is confusing and bad for maintenance.
+
+## Example
+
+Given:
+
+```Ruby
+class Car
+  private
+  def self.start_the_engine; end
+  def drive; end
+end
+```
+
+`Reek` would emit the following warning:
+
+```
+2 warnings:
+  [3]:Car has the unused private class method `start_the_engine` (UnusedPrivateMethod)
+  [4]:Car has the unused private instance method `drive` (UnusedPrivateMethod)
+```
+
+As you can see above, `Unused Private Method` does differentiate between class
+and instance methods.
+
+Private methods that are called via `dynamic dispatch`
```

Comment:
Let’s keep `<code> formatting` for actual code. I don’t think dynamic dispatch needs any quoting, but if you feel like it does just quote it with quotation marks. :)